tests/conftest.py

```diff
@@ -11,21 +19,21 @@ def isolation(fn_isolation):
 def whale(accounts):
     # Totally in it for the tech
     # Update this with a large holder of your want token (the largest EOA holder of LP)
-    whale = accounts.at("0xd2d2F6a38F3A323Df87346413269cdB62cBDDB71", force=True)
+    whale = accounts.at(convert.to_address(config["wallets"]["lp_whale"]), force=True)
     yield whale
 
 
 # this is the amount of funds we have our whale deposit. adjust this as needed based on their wallet balance
 @pytest.fixture(scope="module")
 def amount():
-    amount = 500e18
+    amount = 5000e18
```

Comment:
```suggestion
    amount = Wei("5_000 ether")
```

Reply:
I'll update in my local env and push the commit from there as I also need to import Wei in the file.